index.html

```diff
@@ -222,16 +239,12 @@ <h1 class="about-me">About me</h1>
         </ul>
       </div>
     </section>
-    <footer class="footer-section">
-      <form
-        class="contact-form"
-        action="https://formspree.io/f/mayvbgrw"
-        method="POST"
-      >
-        <p>
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </p>
+    <section class="contact-section contact-form">
+      <h2>
+        I'm always interested in hearing about new projects, so if you'd like
+        to chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/mayvbgrw" method="POST">
         <input
           type="text"
           name="name"
```

Comment:
- Kindly try to stick as much as possible to the Figma design, which includes:
   - [ ] Adjusting the spacing around this section to match the Figma design.
   - [ ] Using the same font style from the Figma design.
   - [ ] Adding a horizontal line above the social media icons.
   - [ ] Adjusting the width of the "Get in touch" button's width to match the Figma design.

| Your design | Figma design | 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/159921788-c1594ecb-c8b1-4ef4-b1bd-09931f686f2a.png) | ![image](https://user-images.githubusercontent.com/62290545/159921909-85790cda-533d-4d26-8dd3-d4049efc4457.png) |